<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:	#E6E6FA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">background-color:#5F9EA0</string>
     </property>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="judul">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PENYEWAAN GEDUNG RATO EBU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>621</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#f3f3f3;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_data">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Data Penyewa</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>621</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="Ok">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>500</y>
         <width>601</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="tggl">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_tggl">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>10</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#00BFFF; border-radius:7px</string>
       </property>
       <property name="text">
        <string>Cari Tanggal</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>321</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#f3f3f3</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>320</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_nama">
          <property name="text">
           <string>Nama Lengkap Penyewa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nama"/>
        </item>
        <item>
         <widget class="QLabel" name="label_no">
          <property name="text">
           <string>Nomor Telepon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="notel"/>
        </item>
        <item>
         <widget class="QLabel" name="label_alamat">
          <property name="text">
           <string>Alamat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="alamat"/>
        </item>
        <item>
         <widget class="QLabel" name="label_acara">
          <property name="text">
           <string>Nama Acara</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="acara"/>
        </item>
        <item>
         <widget class="QLabel" name="label_waktu">
          <property name="text">
           <string>Waktu</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="siang">
            <property name="text">
             <string>Siang (06:00-14:00)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="malam">
            <property name="text">
             <string>Malam (15:00-23:00)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="ex">
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="spasi2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tgl">
          <property name="text">
           <string>Tanggal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2019</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
          <property name="date">
           <date>
            <year>2019</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="spasi1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="add">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>500</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#00BFFF; border-radius:7px</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="dlt">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>500</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FF6347; border-radius:7px</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">background-color:#5F9EA0</string>
     </property>
     <attribute name="title">
      <string>Persyaratan</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PERSYARATAN</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>931</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#f3f3f3; border-radius:5px</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;1.       Pemesanan dikenakan uang muka sewa yang ditetapkan sebesar 30% dari tarif sewa Aula/Ruang Meeting.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;2.       Pelunasan uang sewa secara keseluruhan paling lambat 7 hari sebelum hari pelaksanaan.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;3.       Apabila terjadi pembatalan dari pihak penyewa maka uang muka TIDAK dapat diminta kembali. Namun jika pembayaran sudah dilunasi seluruhnya atau terbayar sebagian, maka uang dikembalikan sebesar 50 % dari nilai uang yang telah dibayarkan (diluar uang muka)&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;4.       Jika pembatalan reservasi dilakukan dengan mengganti pemesanan/reservasi dilain hari, maka dikenakan pinalti pembayaran 15% dari tarif sewa Aula/Ruang Meeting&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;5.       Apabila pembatalan dari pihak pengelola maka penyewa berhak mendapat kompensasi 3 kali uang muka nilai uang muka (termasuk uang muka yang telah dibayarkan).&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;6.       Pemakai gedung harus menggunakan jasa rekanan yang telah terdaftar pada pengelola, jika tidak maka akan dikenakan charge sebesar 10 %&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;7.       Waktu penggunaan gedung serba guna Rato Ebhu&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;8.       Siang : 06.00 s/d 14.00 WIB Malam,15.00 s/d 23.00 WIB.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;9.       Apabila penyewa melebihi waktu yang di tentukan, dikenakan biaya tambahan 10 % dari sewa ruangan dengan batas maksimal 1 jam&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;10.   Penggunaan jasa foto studio / video shoting dan organ tunggal di kenakan charge listrik Rp. 250.000&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;11.   Pada saat waktu adzan/sholat, seluruh kegiatan sound system diharuskan berhenti selama ± 15 menit&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;12.   Pemakaian hiburan dan Organ Tunggal harus menggunakan sound system sendiri&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;13.   Pemasangan media iklan, Reklame, Sponsor dalam bentuk Umbul-umbul, Spanduk, Poster, Banner dsb, sehingga menimbulkan beban pajak, maka menjadi tanggung jawab penuh pihak penyewa.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;14.   Pemasangan dekorasi tidak diperbolehkan memaku, menempelkan pada tembok/pintu Dll. Pengelola berhak meminta ganti rugi yang sesuai kepada penyewa yang mengotori/merusak gedung.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;15.   Dilarang membawa senjata tajam/api serta minuman keras ke dalam area gedung serba guna&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:10pt;&quot;&gt;16.   Pengelola tidak bertanggung jawab atas segala kehilangan barang/kendaraan pengunjung selama kegiatan berlangsung.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true">background-color:#5F9EA0</string>
     </property>
     <attribute name="title">
      <string>Gambar</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>POTRET GEDUNG</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>941</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#f3f3f3; border-radius : 5px</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>281</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Serambi Depan</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/serambi depan.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>240</y>
         <width>281</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Area Parkir</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/area parkir.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>281</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Serambi Resepsionis</string>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/serambi resepsionis.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>0</y>
         <width>281</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Back Drop Aula</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/back drop aula.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>281</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Plasa Gedung </string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/plasa 1.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>240</y>
         <width>281</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Plasa Gedung</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/plasa 2.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
